hack/namespace_scoped.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+# This script generates the contents for a kubeconfig file for a given ServiceAccount. 
+
+# Default the control variables, but they can be overwritten by passing in environment variables.
+CLUSTER_NAME=${CLUSTER_NAME:-bpfman-deployment}
+NAMESPACE=${NAMESPACE:-acme}
+SERVICE_ACCOUNT=${SERVICE_ACCOUNT:-test-account}
+SECRET_NAME=${CLUSTER_NAME:-test-account-token}
```

Comment:
I'm getting this error when running this script:
```
nfvsdn-04:~/go/src/github.com/bpfman/bpfman-operator$ ./hack/namespace_scoped.sh
Error from server (NotFound): secrets "bpfman-deployment" not found
```

However, if I hardcode `SECRET_NAME=test-account-token` here it works.

Reply:
I see the problem. Should have been:
```suggestion
SECRET_NAME=${SECRET_NAME:-test-account-token}
```
I'll push a fix in a few.

Reply:
Done.